clang/include/clang/Driver/Action.h

```diff
@@ -698,6 +698,14 @@ class OffloadWrapperJobAction : public JobAction {
   // Get the compilation step setting.
   bool getCompileStep() const { return CompileStep; }
 
+  // Set the individual wrapping setting.  This is used to tell the wrapper job
+  // action that the wrapping (and subsequent compile step) should be done
+  // with for-each instead of using -batch.
+  void setIndividualWrap(bool SetValue) { IndividualWrap = SetValue; }
```

Comment:
```suggestion
  void setWrapIndividualFiles() { WrapIndividualFiles = true; }
```

I don't think we will ever set it to false anywhere. Also, this will require subsequent changes.

Thanks